bin/release.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+export VERSION_NUMBER=$(jq --raw-output .version package.json)
+export VERSION_TAG="v$VERSION_NUMBER"
+export LATEST_TAG=$(curl -H \
+  "Authorization: token $GITHUB_TOKEN" \
+  https://api.github.com/repos/nasa/cumulus-message-adapter-js/tags | jq --raw-output '.[0].name')
+
+if [ "$VERSION_TAG" != "$LATEST_TAG" ]; then
```

Comment:
This should really be checking if the VERSION_TAG is greater than the LATEST_TAG, but that seems pretty complicated to do for semver tags?

Reply:
https://www.npmjs.com/package/semver-compare, as it's running in a node 8 (soon to be 10 anyway) container?

Reply:
Don't know that we *have* to do this.

Reply:
Yeah, just thinking about it, if `VERSION_TAG` was somehow less than `LATEST_TAG`, it should just mean that the script basically does nothing. The tag creation and release creation would fail and fall back to echo statements. So I'll probably leave as is and avoid the extra dependency.